AppCenterDataStorage/AppCenterDataStorage/AppCenterDataStorage.h

```diff
@@ -1,5 +1,5 @@
 #import <Foundation/Foundation.h>
```

Comment:
General Comments:
there is a tool that you can use to format the code:
1-install:  `brew install clang-format`
2-Run: (form root directory of AppCenter-SDK-Apple) `sh ./clang-format-changed-files.sh`

this will format all the files

Reply:
Yeah, I've been running the format tool!